lib/CMakeLists.txt

```diff
@@ -16,14 +16,14 @@
 #
 ###############################################################################
 
-set(LIB_SRCS type.cpp variable.cpp expr.cpp stmt.cpp gen_policy.cpp sym_table.cpp program.cpp options.cpp)
+set(LIB_SRCS type.cpp ir_value.cpp)
 
-set(SRCS ${LIB_SRCS} main.cpp self-test.cpp)
+set(SRCS ${LIB_SRCS} main.cpp)
 
 add_executable(yarpgen ${SRCS})
 
 target_compile_features(yarpgen PRIVATE cxx_std_14)
-target_compile_definitions(yarpgen PRIVATE BUILD_VERSION="${GIT_HASH}" BUILD_DATE="${BUILD_DATE}")
+#target_compile_definitions(yarpgen PRIVATE BUILD_VERSION="${GIT_HASH}" BUILD_DATE="${BUILD_DATE}")
```

Comment:
We'll need git hash / build date / build compiler / Debug/Release in help message anyway. Make sense adding it from the every beginning.